src/js/common/components/NavButtons.jsx

```diff
@@ -88,10 +89,17 @@ export default class NavButtons extends React.Component {
     );
 
     let buttons;
+    let privacyAgreement;
     if (path === '/review-and-submit') {
       let submitButton;
       let submitMessage;
 
+      privacyAgreement = (
+        <PrivacyAgreement
```

Comment:
I'm not thrilled that this is in the NavButtons component, but it's the only way to position it in the right place at the moment.